src/status_im/contexts/wallet/wallet_connect/modals/send_transaction/view.cljs

```diff
@@ -114,6 +114,6 @@
          :container-style style/data-item
          :title           (i18n/label :t/est-time)
          :subtitle        (if estimated-time
-                            (i18n/label :t/time-in-mins {:minutes estimated-time})
+                            (i18n/label :t/time-in-sec {:seconds (str estimated-time)})
```

Comment:
Just a quick question, do we really need `str` or will `i18n/label` handle numbers just fine? My impression was that we could remove `str` here and in other similar usages.